pkg/metadata/inventories/README.md

```diff
@@ -56,6 +56,18 @@ The payload is a JSON dict with the following fields
     version, ...). This defaults to `"undefined"` when not installed through a tool (like when installed with apt, source
     build, ...).
   - `install_method_installer_version` - **string**:  The version of Datadog module (ex: the Chef Datadog package, the Datadog Ansible playbook, ...).
+  - `feature_security_enabled` - **string**: Either `"true"` or `"false"` depending if the Security Agent is enabled
```

Comment:
The Security Agent is shared by both :
 - Cloud Workload Security (CWS), previously known as Runtime Security
 - Cloud Security Posture management (CSPM), previously known as Compliance

Should we name it `feature_cws_enabled` and `feature_cspm_enabled` ?

Reply:
Are both enabled by the same configuration `runtime_security_config.enabled` ? if not could you gave me the right configuration to look up for each ?